### PR TITLE
docs: update to cloud question on moving clusters

### PR DESCRIPTION
**Pull request by stevenGravy**

### CODE REVIEW

File: docs/pages/choose-an-edition/teleport-cloud/faq.mdx [hunk 1]
```diff
@@ -25,9 +25,9 @@ Yes. Customers can subscribe to Teleport Enterprise Cloud updates at [status.tel
 
 Please [reach out to sales](https://goteleport.com/signup/enterprise) to discuss pricing.
 
-## If I start with Teleport Enterprise Cloud, can I move to Teleport Enterprise or Teleport Open Source, or do I need to start again?
+## If I start with Teleport Enterprise Cloud, can I move to Teleport Enterprise or Teleport Open Source?
 
-If you plan to use S3 and DynamoDB as storage backends, we can provide data for you to import. But you should reach out to us first. If you use a different backend, you will need to start over.
+If you plan to use S3 and DynamoDB as storage backends, we can provide data for you to import. If you use a different backend, you can reuse configurations in a new Teleport cluster. Prior to any planned migration we recommend contacting Teleport to get the best options.
```
Comment (zmb3) [inline docs/pages/choose-an-edition/teleport-cloud/faq.mdx:30]: This isn't clear to me.

By _reuse configuration_ do you mean you can use the same configuration but will have to manually migrate the data?

Comment (stevenGravy) [reply docs/pages/choose-an-edition/teleport-cloud/faq.mdx:30]: Yes, some of the configurations can be reused directly (roles) and others will require some changes.  That will require a new cluster but does not require starting completely over. Starting completely over sounds much starker then the reality.

Comment (zmb3) [reply docs/pages/choose-an-edition/teleport-cloud/faq.mdx:30]: I think you need to find a way to add that to the PR. As-is, this PR leaves me with more questions than answers.

**Comment by alexfornuto** [comment]
@stevenGravy Just checking in since this PR hasn't been updated since Zac's review a few weeks ago.

File: docs/pages/choose-an-edition/teleport-cloud/faq.mdx (hunk 1 above)
Comment (ptgott) [reply docs/pages/choose-an-edition/teleport-cloud/faq.mdx:30]: @stevenGravy Does "configurations" mean "Teleport configuration resources" here?

**Comment by ptgott** [comment]
@stevenGravy Checking to see if you still plan to move forward with this one. Thanks!

**Comment by ptgott** [comment]
Hi @stevenGravy, friendly nudge on this PR. Thanks!

**Comment by zmb3** [comment]
Closing due to inactivity.

**Pull request closed without merging**